### PR TITLE
Add Chromium versions for api.DOMTokenList.toggle.force_argument

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `toggle.force_argument` member of the `DOMTokenList` API, based upon commit history and date.

Commit: https://source.chromium.org/chromium/chromium/src/+/7f56e8d21f5ca85c05c41251c28dfdb4a71bca0a

### CODE REVIEW

**Comment by foolip** [comment]
https://source.chromium.org/chromium/chromium/src/+/7f56e8d21f5ca85c05c41251c28dfdb4a71bca0a is https://trac.webkit.org/changeset/131408/webkit from before the Blink fork, and even though https://www.chromium.org/developers/calendar has branch dates for 2012, there will still have been a delay from when somethings lands in WebKit to when it's rolled into Chromium. So I'd suggest instead looking at the WebKit version for the revision. That's also not entirely reliable, but if the version is *between* the WebKit versions of two browser releases (not equal to either) then one can be pretty confident the change was in the second of those releases but not the first.

From https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=131408 the version was 537.16.0, which would suggest this was in Chrome 24 (537.17) but not Chrome 23 (537.11). Based on the same, it should be in Safari 6.1, which the data already suggests.

Just to verify the method, could you verify by testing?

**Comment by foolip** [comment]
Rather than going deeper into source code archeology, I checked what the argument does and wrote http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8808 as a test. Running on Chrome 23, 24 and 25, I can see that this changed in Chrome 24, not 25.

File: api/DOMTokenList.json [hunk 1]
```diff
@@ -772,10 +772,10 @@
             "description": "<code>force</code> argument",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "25"
```
Comment (foolip) [inline api/DOMTokenList.json:775]:
```suggestion
                "version_added": "24"
```

Per https://github.com/mdn/browser-compat-data/pull/8482#issuecomment-759483886

File: api/DOMTokenList.json [hunk 2]
```diff
@@ -772,10 +772,10 @@
...
               },
               "chrome_android": {
-                "version_added": true
+                "version_added": "25"
```
Comment (foolip) [inline api/DOMTokenList.json:778]:
```suggestion
                "version_added": "24"
```

**Review by foolip — APPROVED** [review]

File: api/DOMTokenList.json (hunk 2 above)
Comment (Elchi3) [reply api/DOMTokenList.json]: There was no chrome_android 24. This broke the CI.

Comment (foolip) [reply api/DOMTokenList.json]: Sorry, I was too quick to merge after applying my own suggestion. This is the second time I've done that I think. I'll try to make it a habit, but branch protection would really great too to make it impossible :)

Comment (Elchi3) [reply api/DOMTokenList.json]: cc @ddbeck is branch protection something you can enable?

Comment (ddbeck) [reply api/DOMTokenList.json]: There's some subtleties to overcome, but yes. See #7171 for details. I'll make it my Friday project, or sooner, depending on how long it takes me to do the release today.

**Pull request merged**